ASPNetCoreMastersTodoList.API/Controllers/ItemsController.cs

```diff
@@ -19,19 +21,46 @@ public ItemsController()
             this._itemService = new ItemService();
         }
 
+        [HttpGet("/items")]
```

Comment:
Since we have added [Route("[controller]")] in the controller level, there is no need to indicate "items" on the route template. It would be more appropriate to change this line to [HttpGet] instead. Similarly, it should be applied to the other routes as well. Indicating "/" as first character in the route template will ignore the route on the controller level.

Reply:
applied on session 5 homework, thank you